lib/block-editor-settings.php

```diff
@@ -76,6 +76,13 @@ function gutenberg_get_block_editor_settings( $settings ) {
 		}
 	}
 
+	// Could house also `__unstableIsBlockBasedTheme` as 'isBlockTheme' => $current_theme->is_block_theme().
+	$current_theme            = wp_get_theme();
```

Comment:
Using `wp_get_theme` to get full `WP_Theme` object in case folks want to use [other helpful methods](https://developer.wordpress.org/reference/classes/wp_theme/).